src/events/EventsSection.tsx

```diff
@@ -100,6 +179,35 @@ export const EventsSection = () => {
     </>
   );
 
+  const clearSelection = (e: any) => {
+    e.stopPropagation();
+    setSelectedEvents([]);
+    setValue("eventTypes", []);
+  };
+
+  const deleteIndividualSelection = (e: any, chip: string) => {
+    const updatedEventSelection = selectedEvents.filter((obj) => obj !== chip);
+    setSelectedEvents(updatedEventSelection);
+    setValue("eventTypes", updatedEventSelection);
+    e.stopPropagation();
+  };
+
+  const chipGroupComponent = () => {
+    return (
+      <ChipGroup>
+        {(selectedEvents || []).map((chip, index) => (
+          <Chip
+            isReadOnly={index === 0 ? true : false}
```

Comment:
The `index === 0` statement already produces a boolean value, so this ternary operation is not needed.
```suggestion
            isReadOnly={index === 0}
```

Reply:
true, I should have spent some time refactoring the obvious code bits like this one! thanks